core-web/libs/sdk/client/src/lib/editor/sdk-editor.ts

```diff
@@ -43,181 +25,12 @@ export function updateNavigation(pathname: string) {
-                // Here extract dot-container from contentlet if is Headless
-                // or search in parent container if is VTL
-                target.dataset?.['dotContainer']
-                    ? JSON.parse(target.dataset?.['dotContainer'])
-                    : getClosestContainerData(target),
-            contentlet: {
-                identifier: target.dataset?.['dotIdentifier'],
-                title: target.dataset?.['dotTitle'],
-                inode: target.dataset?.['dotInode'],
-                contentType: target.dataset?.['dotType']
-            }
-        };
-
-        postMessageToEditor({
-            action: CUSTOMER_ACTIONS.SET_CONTENTLET,
-            payload: {
-                x,
-                y,
-                width,
-                height,
-                payload: contentletPayload
-            }
-        });
-    };
-
-    document.addEventListener('pointermove', pointerMoveCallback);
-
-    subscriptions.push({
-        type: 'listener',
-        event: 'pointermove',
-        callback: pointerMoveCallback
-    });
-}
-
-/**
- *
- *
- * @private
- * @memberof DotCMSPageEditor
- */
-function scrollHandler() {
-    const scrollCallback = () => {
-        postMessageToEditor({
-            action: CUSTOMER_ACTIONS.IFRAME_SCROLL
-        });
-    };
-
-    window.addEventListener('scroll', scrollCallback);
-
-    subscriptions.push({
-        type: 'listener',
-        event: 'scroll',
-        callback: scrollCallback
-    });
-}
-
-/**
- * Listens for changes in the content and triggers a custom action when the content changes.
- *
- * @private
- * @memberof DotCMSPageEditor
- */
-function listenContentChange() {
-    const observer = new MutationObserver((mutationsList) => {
-        for (const { addedNodes, removedNodes, type } of mutationsList) {
-            if (type === 'childList') {
-                const didNodesChanged = [
-                    ...Array.from(addedNodes),
-                    ...Array.from(removedNodes)
-                ].filter(
-                    (node) => (node as HTMLDivElement).dataset?.['dotObject'] === 'contentlet'
-                ).length;
-
-                if (didNodesChanged) {
-                    postMessageToEditor({
-                        action: CUSTOMER_ACTIONS.CONTENT_CHANGE
-                    });
-                }
-            }
-        }
-    });
-
-    observer.observe(document, { childList: true, subtree: true });
-
-    subscriptions.push({
-        type: 'observer',
-        observer
-    });
-}
-
-/**
- * Sends a ping message to the editor.
- *
- */
-export function pingEditor() {
-    postMessageToEditor({
-        action: CUSTOMER_ACTIONS.PING_EDITOR
-    });
-}
-
 /**
  * Checks if the code is running inside an editor.
  * @returns {boolean} Returns true if the code is running inside an editor, otherwise false.
  */
 export function isInsideEditor() {
-    if (window?.parent === window) {
+    if (window.parent === window) {
```

Comment:
The concept of pinging the editor was to know if we were inside the editor. Should we unify both functions of use the ping for this action?

cc: @fmontes

Reply:
@KevinDavilaDotCMS this is true... and by doing this we're removing that and is like going backward because technically just by puting this page in an iframe make it "inside editor", create an issue to figure it out.